docs/RHEL8_docker_install.md

```diff
@@ -0,0 +1,33 @@
+# RHEL8 docker installation
+
+_These instructions are adapted from the article found [here](https://linuxconfig.org/how-to-install-docker-in-rhel-8)_.
+
+### Instructions
+
+First, we need to add and enable the docker-ce repo.
+ ````
+  $ sudo dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
```

Comment:
extra spaces here

Reply:
We should add extra spaces here? Or remove them?

Reply:
remove them please (as it is on [line 15](https://github.com/AICoE/s2i-custom-notebook/pull/36/files#diff-68fc69236c9bc0fa4fcffae3c5289c1dR15) for example

Reply:
Alright, spaces removed. :smiley:   Can I ask though, is this just for consistency, is there a style format we should be following for markdown files? or is there another reason?

Reply:
Well, there's currently not much of a strict single standard to follow, however there are few linters that can help with that - and all of them implement the same set of rules - for example [markdownlint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md) which is an implementation of https://cirosantilli.com/markdown-style-guide/ .

When it comes to this particular change, the only reason, why it mattered to me is consistency. Everything inside 3 backticks ` ``` ` is rendered including relative indent to the backticks:

````
```
abc
```
````
would render to
```
abc
```

In comparison, when indent is present it'll look like this:

````
```
    abc (4 spaces before)
```
````

which results in
```
    abc (4 spaces before)
```

Another aspect is, when you'd want to copy and paste such code, you would copy also the leading spaces - resulting in skipping bash history for example.

Also a cool thing to know (a "fun fact", I'm getting old, lol): code blocks start with 3 backticks -  ` ``` `. But you can use more than 3 to escape backticks inside the code block. For example this block starts with 5 to show how to render 4 and 3 backtick lines
`````
````
```
````` 

On a unrelated note, there are few interesting rules in the standard, that makes the code blocks a lot easier to use, like for example [MD014](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md014)

Reply:
:pray: Thanks! Also, I now see you meant the space **on** the line, not between the lines (duh) :smile: 

I'll fix this now.

Reply:
huh, sorry for that misunderstanding. :smile: I could have used more than 3 words, right? :smile: